tools/scylla-nodetool.cc

```diff
@@ -1083,7 +1091,7 @@ void viewbuildstatus_operation(scylla_rest_client& client, const bpo::variables_
         fmt::print("{}.{} has not finished building; node status is below.\n", keyspace, view);
         fmt::print("\n");
         table.print();
-        // TODO: should return with status code of 1 in this case
+        throw operation_failed_with_status(1);
```

Comment:
I was trying to come up with distinct return codes for different class of errors. Returning 1 from here, or in general allowing the operation to return an arbitrary code will upset this.

I don't know how valuable it is to have these distinct error codes to begin with, but if you think they could be useful, I propose the following:
* Merge this new exception with the existing `operation_failed_on_scylladb` and have this always trigger returning 1.
* Reshuffle the existing codes such that no other error class returns 1.

WDYT?

Reply:
ok, we can go with this and once we have all command I'll try to consolidate it.

Reply:
sure. thanks!